### ISSUE TITLE
[Feature]: Combine Emily Integration Tests with Standard Integration tests

### ISSUE THREAD

**Issue by AshtonStephens** [stacks-network/sbtc#505]
# Feature - Combine Emily Integration Tests with Standard Integration tests

## 1. Description

Combine Emily Integration tests with the standard integration tests so the testing isn't disjointed.

### 1.1 Context & Purpose
The emily integration tests are separate from the main integration tests, which leads to an odd and inefficient testing setup. This ticket is to combine the integration tests for all the components so there's a single integration environment.

## 2. Technical Details:
All the integration environments are in a single docker compose.

### 2.1 Acceptance Criteria:
   - [ ] The emily containers spin up with the rest of the containers
   - [ ] The emily server runs in a container
   - [ ] The emily integration tests run in CI

## 3. Related Issues and Pull Requests (optional):
N/A

**Comment by AshtonStephens** [comment]
Done with #678

**Comment by matteojug** [comment]
Was this completely done with 678? Given that in there the was a TODO referring this (https://github.com/stacks-network/sbtc/pull/678/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R105).

Also the original ticket was talking about moving Emily integration setup in the same docker compose as the other integration tests, was that still relevant?

**Comment by AshtonStephens** [comment]
Yep, I mixed these up and I should have linked #680 instead, reopening until that's merged.
